src/main/java/com/meetup/server/auth/application/CustomOAuth2UserService.java

```diff
@@ -0,0 +1,57 @@
+package com.meetup.server.auth.application;
+
+import com.meetup.server.auth.dto.CustomOAuth2User;
+import com.meetup.server.auth.dto.response.OAuthAttributes;
+import com.meetup.server.member.domain.Member;
+import com.meetup.server.member.persistence.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.client.userinfo.DefaultOAuth2UserService;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserRequest;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserService;
+import org.springframework.security.oauth2.core.OAuth2AuthenticationException;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.Map;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CustomOAuth2UserService implements OAuth2UserService<OAuth2UserRequest, OAuth2User> {
+
+    private final MemberRepository memberRepository;
+
+    @Override
+    public OAuth2User loadUser(OAuth2UserRequest userRequest) throws OAuth2AuthenticationException {
+
+        OAuth2UserService<OAuth2UserRequest, OAuth2User> delegate = new DefaultOAuth2UserService();
```

Comment:
```suggestion
        private static final OAuth2UserService<OAuth2UserRequest, OAuth2User> delegate = new DefaultOAuth2UserService();
```

소셜 로그인할 때마다 DefaultOAuth2UserService 객체가 생성되고 있어서 상수로 선언해도 좋을 거 같아요!